cypress/support/fragments/finance/funds/funds.js

```diff
@@ -262,5 +263,33 @@ export default {
     cy.do([
       Accordion({ id: 'currentBudget' }).find(MultiColumnListCell({ content: fundCode.concat('-', fiscalYear) })).click()
     ]);
+  },
+
+  seletBudgetDetails:(rowNumber = 0) => {
```

Comment:
should it be selectBudgetDetails name?

Reply:
Yeah,I open the budget information in the following tab

Reply:
Tetiana means that you missed a letter in the name of the function: seletBudgetDetails

Reply:
done